api/src/controllers/internal/Execution.ts

```diff
@@ -29,6 +29,7 @@ interface ExecuteFileParams {
   session?: Session
   runTime: RunTimeType
   forceStringResult?: boolean
+  addPrintOutput?: boolean
```

Comment:
its better to move this param to `ExecuteProgramParams`

Reply:
Done